tools/mydumper.md

```diff
@@ -0,0 +1,46 @@
+---
+title: mydumper Instructions 
+summary: Use mydumper to export data from TiDB.
+category: tools
+---
+
+# mydumper Instructions
+
+## What is mydumper?
+
+`mydumper` is a fork of the [mydumper](https://github.com/maxbube/mydumper) project with additional functionality specific to TiDB. It is the recommended method to use for logical backups of TiDB.
+
+[Download the Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz).
+
+## What enhancements does this contain over regular mydumper?
+
++ Use `tidb_snapshot` to provide backup consistency instead of `FLUSH TABLES WITH READ LOCK`
+
++ Include the hidden `_tidb_rowid` column in exports when present
```

Comment:
This affects both the CREATE TABLE schema and INSERT INTO data files?

```sql
CREATE TABLE tbl_name (
    _tidb_rowid BIGINT UNSIGNED AUTO_INCREMENT PRIMARY KEY,  -- there will be this?
    ...
) ...;

INSERT INTO tbl_name VALUES (1, ...), (2, ...), ...;
--                           ^ there will be this?
```

Reply:
It is only present in INSERT, not CREATE TABLE.  I will make it clearer.